daisy_workflows/kernel_commit_query/debian.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -xeu
+
+apt-get update -y && apt-get upgrade -y && apt-get install -y git linux-source
```

Comment:
since you are using debian to get to its package infrastructure and you aren't necessarily using anything that is specific to the image, you might consider running this directly in a container environment instead of through a daisy workflow. our container images should be just as capable of downloading the `linux-source` package for a given distribution and running python code against it

Reply:
We synced off line, in the "apply & check" strategy we are relying on the currently running kernel.